hackathon_site/dashboard/frontend/src/components/dashboard/EditTeam/EditTeam.tsx

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+import Alert from "@material-ui/lab/Alert";
+
+import FileCopyIcon from "@material-ui/icons/FileCopy";
+import SideSheetRight from "components/general/SideSheetRight/SideSheetRight";
+
+import styles from "./EditTeam.module.scss";
+import { isTeamModalVisibleSelector, closeTeamModalItem } from "slices/ui/uiSlice";
+import { useDispatch, useSelector } from "react-redux";
+import { Grid } from "@material-ui/core";
+import { maxTeamSize } from "../../../constants";
+
+interface TeamModalProps {
+    teamCode: string;
+    canChangeTeam: boolean;
+    teamSize: number;
+}
+
+const TeamInfoBlock = ({ teamCode, canChangeTeam, teamSize }: TeamModalProps) => {
+    return (
+        <div>
```

Comment:
you don't need this extra div here, and since you don't have any code before the return statement you can remove that too

```
const TeamInfoBlock = (...) => (
    <Alert ...>
         ....
    </Alert>
)
```